delivery-kube-config-files/internal-content-preview-api.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         - name: CONTENT_SOURCE_URI
           value: http://content-public-read-preview:8080/content-preview/
         - name: INTERNAL_COMPONENTS_SOURCE_URI
-          value: http://internal-components-preview:8080/internalcomponents/content-preview/
+          value: http://internal-components-preview:8080/internal-components-preview/content-preview/
```

Comment:
Shouldn't the URL be _http://internal-components-preview:8080/content-preview/_?
The service is using a docker image created from https://github.com/Financial-Times/content-preview and the endpoint it exposes is /content-preview/{uuid}.